Fixed auto update, Trade site load, Hotkey handling

rhzk opened the pull request:
* General
  * Auto update now downloads zip and tells user to extract it manually (windows only)
  * Restores users old clipboard after checking a item
  * Reduced load on trade website a lot
  * Added code block for item data and some other useful error information 
* Hotkeys
  * When a hotkey is pressed program ignores any other hotkeys until that one is handled
  * Exiting is now a hotkey instead of keyboard interrupt (default alt+m, alt+d is apparently a keyboard interrupt)

Ethck committed:
Timeout parameters adjusted due to timeouts occurring too rapidly.

Ethck committed:
Adjusted poeprices.info timeout.

Ethck committed:
Fixed conflict in web.py.

Ethck committed:
Updated version to v0.95

rhzk committed:
Fixed auto update

rhzk committed:
Fixed auto update download zip for windows

Ethck commented:
I think it is easier & better for the user if we still download the latest version, we just need to tell them to manually unzip it/install it. Thoughts?

rhzk committed:
Fix #194

rhzk commented:
Added back option to download zip on windows, and some info printed out + a timer so people have time to read before the program exits.

Also fixed #194

rhzk committed:
No longer accept new hotkeys while already handling one, reduced load on trade website, exit program is a hotkey now, prints better info for issues

rhzk committed:
Imr to mod parsing
